### PR TITLE
Update community docs

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Update templates
- Remove redundant templates
- Fix broken links
- Update outdated instructions
- Change the order of the content guidelines
- Change `collaboration` folder into `sigs-and-wgs`

**Related issue(s)**
https://github.com/kyma-project/community/issues/556

### CODE REVIEW

File: docs/guidelines/content-guidelines/01-content-strategy.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Select a title that describes the task that's accomplished, not the documented s
 With regards to structure, it’s nice to have an **introductory paragraph** ("why would I want to do this task?"), **prerequisites** if needed, then the **steps** in a numbered list, and finally the expected **result** that shows the operation was successful.
 It's good practice to have 5-9 steps; anything longer can probably be split.
 
-For all step instructions, use the [task topic template](../../../templates/resources/task.md).
+For all step instructions, use the [tutorial  template](https://github.com/kyma-project/community/blob/main/templates/resources/tutorial.md).
```
Comment (mmitoraj) [inline docs/guidelines/content-guidelines/01-content-strategy.md:35]: The link doesn't work. Try: https://github.com/kyma-project/community/blob/main/templates/resources/tutorials.md
However, when I look at the template I would change the **Type** from `Tutorials` to `Tutorial` I guess 🤔

File: docs/guidelines/content-guidelines/01-content-strategy.md [hunk 2]
```diff
@@ -52,15 +52,15 @@ In the title, mention the symptom that needs fixing ("Cannot access...") or the
...
+For all troubleshooting topics, use the [troubleshooting topic template](https://github.com/kyma-project/community/blob/main/templates/resources/troubleshooting.md).
 
 ### Release notes
 
 Release notes announce what's new in Kyma.
 
 After an introductory paragraph, a list briefly presents the new and changed features. Links lead to longer paragraphs that describe the changes in more detail.
 
-For release notes, use the [release note template](../../../templates/resources/release-notes.md).
+For release notes, use the [release note template](https://github.com/kyma-project/community/blob/main/templates/resources/release-notes.md).
```
Comment (mmitoraj) [inline docs/guidelines/content-guidelines/01-content-strategy.md:63]:
```suggestion
For release notes, use the [release notes template](https://github.com/kyma-project/community/blob/main/templates/resources/release-notes.md).
```

File: docs/guidelines/content-guidelines/06-release-notes.md [hunk 3]
```diff
@@ -36,5 +36,5 @@ Whenever a user must perform some manual steps in order to use the newly release
 
 ## Templates
 Here are all the release notes-related templates you may find useful:
-- [Release notes](../../../templates/resources/release-notes.md)
-- [Release notes input](../../../templates/resources/release-notes-input.md)
+- [Release notes](https://github.com/kyma-project/community/blob/main/templates/resources/release-notes.md)
```
Comment (mmitoraj) [inline docs/guidelines/content-guidelines/06-release-notes.md:39]:
```suggestion
- [Release notes template](https://github.com/kyma-project/community/blob/main/templates/resources/release-notes.md)
```

File: docs/guidelines/templates/templates-type.md [hunk 4]
```diff
@@ -12,51 +12,53 @@ This table contains templates for general documents, such as `NOTES.txt` and `RE
...
-| [`examples`](https://github.com/kyma-project/examples) | [`README.md`](https://github.com/kyma-project/community/blob/main/guidelines/templates/resources/example_README.md) | Use the template for examples. | All contributors |
+| [`kyma`](https://github.com/kyma-project/kyma)| [`NOTES.txt`](https://github.com/kyma-project/community/blob/main/templates/resources/NOTES.txt) | Use the template for charts. | All contributors |
+| [`kyma`](https://github.com/kyma-project/kyma)| [`README.md`](https://github.com/kyma-project/community/blob/main/templates/resources/chart_README.md) | Use the template only for internal charts with in-house components. | All contributors |
+| [`examples`](https://github.com/kyma-project/examples) | [`README.md`](https://github.com/kyma-project/community/blob/main/templates/resources/example_README.md) | Use the template for examples. | All contributors |
 | All | [`README.md`](https://github.com/kyma-project/community/blob/main/guidelines/repository-guidelines/repository-template/README.md) | Use the template for the `README.md` document in any repository, directory, or subdirectory within the Kyma organization. | All contributors |
```
Comment (mmitoraj) [inline docs/guidelines/templates/templates-type.md:18]: Please update the link - Maja found it during the last SIG Content :) We also agreed that only one general README.md template would be sufficient.

File: docs/guidelines/templates/templates-type.md [hunk 5]
```diff
@@ -12,51 +12,53 @@ This table contains templates for general documents, such as `NOTES.txt` and `RE
...
+This table lists all document types used to fully describe a Kyma area and/or component.
 
 | Repository | Template | Usage | Author |
 |---|---|---|---|
-| No specific repository | [`Official Kyma graphics and logo`](https://github.com/kyma-project/community/blob/main/guidelines/templates/resources/assets) | Use these graphics for any Kyma-related communication. | All contributors |
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`concept.md`](https://github.com/kyma-project/community/blob/main/templates/resources/concept.md) | Use the template to introduce a given concept. | All contributors |
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`architecture.md`](https://github.com/kyma-project/community/blob/main/templates/resources/architecture.md) | Use the template to describe the architecture of a Kyma component. | All contributors |
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`details.md`](https://github.com/kyma-project/community/blob/main/templates/resources/details.md) | Use the template to provide more details about a Kyma component. | All contributors |
```
Comment (mmitoraj) [inline docs/guidelines/templates/templates-type.md:28]: Could be removed

File: docs/guidelines/templates/templates-type.md [hunk 6]
```diff
@@ -12,51 +12,53 @@ This table contains templates for general documents, such as `NOTES.txt` and `RE
...
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`cli-reference.md`](https://github.com/kyma-project/community/blob/main/templates/resources/cli-reference.md) | Use the template to describe the syntax and the use of CLI commands for a given component. | All contributors |
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`tutorials.md`](https://github.com/kyma-project/community/blob/main/templates/resources/tutorial.md) | Use the template to provide a clear step-by-step instruction that helps the user to understand a given concept better. | All contributors |
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`troubleshooting.md`](https://github.com/kyma-project/community/blob/main/templates/resources/troubleshooting.md) | Use the template to explain all details needed for Kyma and its components' troubleshooting. | All contributors |
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`metrics.md`](https://github.com/kyma-project/community/blob/main/templates/resources/metrics.md) | Use the template to describe custom and default metrics for services or controllers. | All contributors |
```
Comment (mmitoraj) [inline docs/guidelines/templates/templates-type.md:34]: Could be removed

File: docs/guidelines/templates/templates-type.md [hunk 7]
```diff
@@ -12,51 +12,53 @@ This table contains templates for general documents, such as `NOTES.txt` and `RE
...
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`details.md`](https://github.com/kyma-project/community/blob/main/templates/resources/details.md) | Use the template to provide more details about a Kyma component. | All contributors |
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`configuration.md`](https://github.com/kyma-project/community/blob/main/templates/resources/configuration.md) | Use the template to describe configuration options for a Kyma chart or sub-chart. | All contributors |
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`custom-resource.md`](https://github.com/kyma-project/community/blob/main/templates/resources/custom-resource.md) | Use the template to describe a custom resource (CR) and its elements. | All contributors |
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/main/docs) | [`cli-reference.md`](https://github.com/kyma-project/community/blob/main/templates/resources/cli-reference.md) | Use the template to describe the syntax and the use of CLI commands for a given component. | All contributors |
```
Comment (mmitoraj) [inline docs/guidelines/templates/templates-type.md:31]: Could be removed

File: templates/repository-template/.github/ISSUE_TEMPLATE/security-vulnerability.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
+---
+name: Security Vulnerability
```
Comment (mmitoraj) [inline templates/repository-template/.github/ISSUE_TEMPLATE/security-vulnerability.md:2]:
```suggestion
name: Security vulnerability
```

File: templates/resources/cli-reference.md [hunk 9]
```diff
@@ -2,14 +2,11 @@
 title: CLI reference
```
Comment (mmitoraj) [inline templates/resources/cli-reference.md:2]: The doc could be removed or archived

File: templates/resources/details.md [hunk 10]
```diff
@@ -3,17 +3,13 @@ title: {Document title}
 type: Details
```
Comment (mmitoraj) [inline templates/resources/details.md:3]: The document could be removed or archived

File: templates/resources/metrics.md [hunk 11]
```diff
@@ -3,28 +3,26 @@ title: {Service/Controller name}
 type: Metrics
```
Comment (mmitoraj) [inline templates/resources/metrics.md:3]: The document could be removed or archived

File: templates/resources/release-notes.md [hunk 12]
```diff
@@ -16,57 +16,50 @@ redirectFrom:
...
 
 > Add a **CAUTION** at the beginning of the release notes whenever there are any important migration and/or update steps required for users to perform before migrating to the new Kyma version. Link to a separate migration guide under `kyma/docs/migration-guides` in which you provide these steps, describe the changes and reasons behind them, and list potential benefits for the users.  
 
-> Introduce other component features or fixes that are included in the release notes. They should reflect the names of subsections under each component. Add relative links to component sections.
+> After the introductory paragraph, list the highlights of the release as bullet points and provide relative links to their corresponding sections.
 
-- [Application Connector](#application-connector) - {List of other features and fixes}
+- [Application Connectivity](#application-connectivity) - {List of other features and fixes}
```
Comment (mmitoraj) [inline templates/resources/release-notes.md:25]: We could also add API Exposure.

**Review by mmitoraj — COMMENTED** [review]
I added some suggestions. Thank you :)

File: templates/resources/details.md (hunk 10 above)
Comment (klaudiagrz) [reply templates/resources/details.md]: Removed :)

File: templates/resources/metrics.md (hunk 11 above)
Comment (klaudiagrz) [reply templates/resources/metrics.md]: Removed :)

File: templates/resources/release-notes.md (hunk 12 above)
Comment (klaudiagrz) [reply templates/resources/release-notes.md:26]: Added

File: README.md [hunk 13]
```diff
@@ -1,38 +1,54 @@
...
 
-Before you start to contribute, review the [contribution rules](./docs/contributing/02-contributing.md). To understand better the way the Kyma community works, read about the [working model](./docs/governance/01-governance.md) and learn about the [Special Interest Groups and Working Groups](./collaboration/01-sig-and-wg.md) in Kyma.
+Create an issue on [GitHub](https://github.com/kyma-project) to report a bug or request a feature. Go to the given repository's issue tracker and raise an issue using one of the available templates. The owners of the repository will review your issue and provide you with their feedback. If you are not certain which repository your issue refers to, open it in the [`kyma`](https://github.com/kyma-project/kyma) repository. For more information on how issues are handled within the Kma organization, read about the [issues workflow](docs/governance/03-issues-workflow.md).
 
-All contributors follow the recommended [Git workflow](./docs/contributing/03-git-workflow.md) that relies on forks, branches, rebasing, and squashing. Review it to ensure that you contribute high-quality code and content to the project.
+## Special Interest Groups & Working Groups
```
[the remark this answers is not in the document]
Comment (mmitoraj) [reply README.md:21]: We can do that with the follow-up task after Kyma 2.0.

File: templates/resources/release-notes.md [hunk 14]
```diff
@@ -16,57 +16,51 @@ redirectFrom:
...
 
 > Add a **CAUTION** at the beginning of the release notes whenever there are any important migration and/or update steps required for users to perform before migrating to the new Kyma version. Link to a separate migration guide under `kyma/docs/migration-guides` in which you provide these steps, describe the changes and reasons behind them, and list potential benefits for the users.  
 
-> Introduce other component features or fixes that are included in the release notes. They should reflect the names of subsections under each component. Add relative links to component sections.
+> After the introductory paragraph, list the highlights of the release as bullet points and provide relative links to their corresponding sections.
 
-- [Application Connector](#application-connector) - {List of other features and fixes}
+- [API exposure](#api-exposure) - {List of other features and fixes}
```
Comment (mmitoraj) [inline templates/resources/release-notes.md:25]:
```suggestion
- [API Exposure](#api-exposure) - {List of other features and fixes}
```

**Review by mmitoraj — COMMENTED** [review]
I guess only API exposure could be changed to API Exposure. I'm adding the `do-not-merge/hold` label and giving my approval. Once the area name is updated you should be able to remove the label and marge :)

**Review by mmitoraj — DISMISSED** [review]

**Review by m00g3n — APPROVED** [review]
LGTM

**Review by PK85 — APPROVED** [review]

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**